wpeverest/user-registration
# Compatibility - ConvertKit Addon

### All Submissions:






### Changes proposed in this Pull Request:

Test the email marketing addon of ConvertKit (now Kit) by generating v3 keys from [app.kit.com](https://app.kit.com/account_settings/developer_settings) and sync fields and perform action from form settings.

Dependency: https://github.com/wpeverest/user-registration-pro/pull/506
Clone repository: https://github.com/wpeverest/user-registration-convertkit.git

Closes # .

### How to test the changes in this Pull Request:

1. Add Connection from User Registration > Settings > Integration > ConvertKit
2. Form Settings > Kit > Enable and sync fields
3. Register form and check from app.kit dashboard wheather everything is working properly or not.

### Types of changes:




### Other information:




### Changelog entry

> Compatibility - ConvertKit Addon
